code/datums/components/autofire.dm

```diff
@@ -37,7 +38,7 @@
 	auto_fire_shot_delay = _auto_fire_shot_delay
 	burstfire_shot_delay = _burstfire_shot_delay
 	burst_shots_to_fire = _burst_shots_to_fire
-	auto_burst_fire_shot_delay = 3 * auto_fire_shot_delay
+	auto_burst_fire_shot_delay = _auto_burst_fire_shot_delay * auto_fire_shot_delay
```

Comment:
not a fan of this solution, this is less modular than having a value independant of auto_fire_shot_delay.

It's not that bad though, but in that case you need to rename the argument _auto_burst_fire_shot_delay into like auto_burst_fire_shot_multiplicator or something